felix/fv/wireguard_test.go

```diff
@@ -1748,6 +1778,7 @@ func wireguardTopologyOptions(routeSource string, ipipEnabled, wireguardIPv4Enab
 	// Indicate wireguard is enabled
 	topologyOptions.WireguardEnabled = wireguardIPv4Enabled
 	topologyOptions.WireguardEnabledV6 = wireguardIPv6Enabled
+	topologyOptions.WireguardThreadingEnabled = wireguardThreadingEnabled
```

Comment:
You also need to set the env variable to enable/disable the feature. As an example, you need to set
```
topologyOptions.ExtraEnvVars["FELIX_WIREGUARDTHREADINGENABLED"] = "true"
```
The other option is to set it in the felixconfig resource, which is used at the end of this function to enabled wireguard. It's probably use this option, since it's already there.

```
felixconfig.Spec.WiregauardThreadingEnabled = &enabled
```

Not enabling the setting, is the reason, the FV test is failing.
You can check if the FV test is passing locally by running:

```
make fv GINKGO_FOCUS="the Wireguard threading config should be configured property" GINKGO_ARGS="-ginkgo.progress -ginkgo.v -ginkgo.failFast" |& tee fv.log
```

Reply:
Thank you that seems to have fixed the test error: https://gist.github.com/jrcichra/4020410442b84bd6d5d75ccdf93fc29c